docker/caffe2/jenkins/build.sh

```diff
@@ -11,7 +11,7 @@ if [ -z "${image}" ]; then
 fi
 
 UBUNTU_VERSION="$(echo "${image}" | perl -n -e'/ubuntu(\d+\.\d+)/ && print $1')"
-CENTOS_VERSION="$(echo "${image}" | perl -n -e'/centos(\d+)/ && print $1')"
+CENTOS_VERSION="$(echo "${image}" | perl -n -e'/centos(\d+\.?\d*\.?\d*)/ && print $1')"
```

Comment:
Unclear whether this is upstreamable.

Reply:
This was to fix the chance of centos7.5.1804, which originally turns to centos7 . centos7.5 also turns into centos7.  Is this script upstreamed already?

Reply:
Also, we cannot use centos7.5, because that image doesn't exist. Unlike Ubuntu, we need to be more specific using centos7.5.1804. See https://hub.docker.com/_/centos